### ISSUE TITLE
Will there be Vue JS/ Nuxt JS support in the near future?

### ISSUE THREAD

**Issue by HamadaFMahdi** [get-convex/convex-demos#16]
Thank you for this!
If there isn't any planned Vue JS support how can we integrate Convex into a generic frontend project? Or would you have to set up a Node server?

**Comment by thomasballinger** [comment]
Hello! I know people are using Vue with Convex today by wrapping the [BaseConvexClient](https://docs.convex.dev/api/classes/browser.BaseConvexClient), and generally Convex should be able to be used in any fronted framework that lets you use JavaScript libraries. There are no immediate plans for this an official Vue.js client however.

**Comment by KevinLeneschmidt** [comment]
Hello, are there still no plans for that? Currently, we have to use the ConvexClient and use the .onUpdate to enable the realtime feature, right? A client would maybe offer a better solution.

**Comment by thomasballinger** [comment]
That's right, the [`ConvexClient`](https://docs.convex.dev/api/classes/browser.ConvexClient) is the way to go here, new since the above comment in August.

Hopefully this client provides everything necessary to create a Vue/Nuxt-specific client! If there's anything else to add let us know. If you're interested in independently sharing a wrapper you write but are stuck on something let me know, happy to answer questions or pair program a bit.
